### ISSUE TITLE
Is it possible to figure out if the content of two pdf documents with different metadata is the same?

### ISSUE THREAD

**Issue by JanRuettinger** [pdfcpu/pdfcpu#998]
I want to find out if two pdf documents with different metadata have the same content.

I thought about the following approaches:
1. Calculate hash for both pdf documents
=> Doesn't work since the different metadata leads to different hash values

2. Strip the metadata from the pdf document and calculate hash
=> Just removing "Properties" is not enough, hashes are still different
=> I couldn't figure out how to just get the raw content of a pdf

3. Extract the content in a different way and hash it then
=> works well for text based pdfs
=> doesn't work well for image based pdfs (becomes more tricky)

Is it possible to extract the raw content without any metadata somehow?

**Comment by hhrutter** [comment]
It really comes down to your definition of `Content`.
Eg. what you see is not always what's there because of transparency and so forth.
Only in a closed environment, where I produce all the files, I may be relying on the `ID` field in the trailer.

**Comment by JanRuettinger** [comment]
Yes I think content stream is the right abstraction level. How can I access the content streams with pdfcpu?

**Comment by hhrutter** [comment]
The following returns a byte slice of the page content for a page dict:
`func (xRefTable *XRefTable) PageContent(d types.Dict) ([]byte, error)`
